bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_options.html.erb

```diff
@@ -50,7 +50,7 @@ end
             }, class: "focus:ring-blue h-4 w-4 text-blue border-slate-300 rounded" %>
           </div>
           <div class="ml-2.5 text-sm pr-4">
-            <%# TODO: should be localized %>
+            <%= t("global.bulk_select.all") %>
             All
```

Comment:
I think we need to remove the hard coded `All` on line 54 here.

Reply:
:man_facepalming: Just deleted it!